### ISSUE TITLE
Narrator announcing 'Collapsed' on Button components

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-windows#11734]
### Problem Description

When focused on a Button component with Narrator on, Narrator announces it's state as collapsed as part of the accessibility description i.e on the 'Button with default styling' on the Button page in the RNTester app, Narrator announces 'Press to submit application, button, collapsed', even though a Button is not expandable/collapsible. 

### Steps To Reproduce

1. Load RNTester in Playground
2. Turn Narrator on
3. Navigate to the Button component page
4. Focus on any of the buttons and wait for the Narrator announcement

### Expected Results

Narrator announces 'Collapsed' at the end of the announcement for a Button component.

### CLI version

12.0.0-alpha.3

### Environment

```markdown
System:
  OS: Windows 10 10.0.22621
  CPU: "(24) x64 AMD Ryzen Threadripper PRO 3945WX 12-Cores     "
  Memory: 46.58 GB / 63.86 GB
Binaries:
  Node:
    version: 18.12.1
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.19
    path: C:\Program Files (x86)\Yarn\bin\yarn.CMD
  npm:
    version: 8.19.2
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
    AllowAllTrustedApps: Enabled
    Versions:
      - 10.0.18362.0
      - 10.0.19041.0
      - 10.0.22000.0
      - 10.0.22621.0
IDEs:
  Android Studio: AI-213.7172.25.2113.9123335
  Visual Studio:
    - 17.5.33530.505 (Visual Studio Enterprise 2022)
    - 16.11.33529.622 (Visual Studio Enterprise 2019)
Languages:
  Java:
    version: 18.0.2.1
    path: C:\Program Files\Common Files\Oracle\Java\javapath\javac.EXE
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react: Not Found
  react-native: Not Found
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Target Platform Version

None

### Visual Studio Version

Visual Studio 2022

### Build Configuration

Debug

### Snack, code example, screenshot, or link to a repository

Example of ExpandCollapsePattern value on UIA tree for button on RNTester page:
![image](https://github.com/microsoft/react-native-windows/assets/30995198/a296f186-6f86-4cc9-a3ca-4e2cf4a19727)

**Comment by chrisglein** [comment]
May have something to do with going between no accessibility properties set to having one set and then having unexpected defaults on the peer. @jonthysell was looking into how this was handled in Paper vs. Fabric recently.

**Comment by chrisglein** [comment]
@YajurG You've merged the PR but this issue still seems active. Looking at the history, I think that's because the development link was created after the PR was already merged. Note that the template says "Resolves [Add Relevant Issue Here]" but you don't want to include the brackets there (so "Resolves #11734", not "Resolves [#11734]" - it's picky about the syntax). Anyway, safe to close now?

**Comment by YajurG** [comment]
Oops yes my bad - this issue should be safe to close. I think I linked the PR that would close this issue after it had already been closed but this one should be resolved.
